Makefile

```
@@ -471,7 +471,7 @@ GINKGO ?= $(LOCALBIN)/ginkgo-$(GINKGO_VERSION)
 KUSTOMIZE_VERSION ?= v5.4.2
 CONTROLLER_TOOLS_VERSION ?= v0.14.0
 ENVTEST_VERSION ?= release-0.17
-GOLANGCI_LINT_VERSION ?= v1.64.8
+GOLANGCI_LINT_VERSION ?= v2.5.0
```

```suggestion
GOLANGCI_LINT_VERSION ?= v2.6.2
```

Can we try updating to the latest version available at this time? https://github.com/golangci/golangci-lint/releases/tag/v2.6.2